spec/jobs/usps_auth_token_refresh_job_spec.rb

```diff
@@ -86,7 +86,7 @@
     end
```

Comment:
I can't select it, but I'd expect the test above to error -- if we're catching a `ServerError`, this should fail:

```ruby
        expect do
          subject.perform
        end.to raise_error
```
because there is no error being thrown

Reply:
i see what you're saying, it is passing though which is curious

Reply:
If you take off the `expect { ... }.to_not raise_error` you see that it's throwing an error for a method expectation based on a different logging method called.

so it's technically still throwing an error and the spec technically passes. 

If the test was originally written to say `expect { ... }.to_not raise_error(Faraday::ServerError)` then it would have been a clearer failure here

Reply:
when i update the test to `...to_not raise_error(Faraday::ServerError)` it still passes...but when I print the error it is a Faraday::ServerError

Reply:
It passes with `to_not raise_error(Faraday::ServerError)` because gets this error instead from the stub
```
       #<InstanceDouble(Analytics) (anonymous)> received unexpected message :idv_usps_auth_token_refresh_job_network_error with ({:exception_class=>"Faraday::ServerError", :exception_message=>"the server responded with status 500"})
```

I pushed 8401797994fe3a7f7e0eb19eef81b8a53ad551f5 to switch out the `instance_double` for a `FakeAnalytics` instance which is a better fake logger and has clearer spec failures when the rescue for `Faraday::ServerError` is removed

Reply:
ahh ok that makes sense! i think we still need the completion log, I'll add it in. Thanks for making this change!